### PR TITLE
🤖 Simplify tests.toml files

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR simplifies the `tests.toml` files by removing any `include = true` properties, which are redundant as the default value of this field is `true`.

As the vast majority of the test cases use `include = true`, omitting it makes the `tests.toml` files more readable and make the exceptional cases (`include = false`) easier to identify.

We've updated `configlet` to use this new behavior. See https://github.com/exercism/configlet/issues/186 and https://github.com/exercism/configlet/issues/207 for the corresponding discussion.

## Tracking

https://github.com/exercism/v3-launch/issues/31

### CODE REVIEW

**Comment by BethanyG** [comment]
@ErikSchierboom  - configlet is failing all the concept stubs for not having an author in `<concep>/.meta/config.json`.  Do I need to add one in?

**Comment by BethanyG** [comment]
@cmccandless  -- looks like we'll need to update `generate_tests.py` for this, since it is looking for `includes`??

**Comment by ErikSchierboom** [comment]
> configlet is failing all the concept stubs for not having an author in <concep>/.meta/config.json. Do I need to add one in? Can we make it a TODO, or does it have to be an actual user?

You could, for now, use "TODO" as the author username. It does mean that if https://github.com/todo/ ever creates an account at exercism.lol, they will be awarded the reputation 😉

**Comment by cmccandless** [comment]
> @cmccandless  -- looks like we'll need to update `generate_tests.py` for this, since it is looking for `includes=True`??   At least that's what it looks like for `filter_test_cases()` starting at line 201...unless I am reading that incorrectly.

I'll take a look to make sure, but I thought I already corrected for this.

**Comment by BethanyG** [comment]
@ErikSchierboom  -- "fixed" this in #2417 (_Corey and I are co-authors on all the stubs now._), so once that's merged, this can be rebased.  Waiting for approval from @cmccandless  on that.

**Review by cmccandless — APPROVED** [review]
I just checked to make sure: our dataclass for tests.toml defaults include to `True`, so this is safe.

https://github.com/exercism/python/blob/main/bin/data.py#L254

**Comment by ErikSchierboom** [comment]
Great!

**Pull request merged**
